modules/dynamic-apiserver/pkg/cli/install_operation.go

```diff
@@ -119,6 +120,7 @@ func makeInstallCmd(k8sclient kubernetes.Interface) *cobra.Command {
 	_ = cmd.PersistentFlags().String(installDockerRegistryURL, "", "Specify private docker registry URL")
 	_ = cmd.PersistentFlags().String(installDockerRegistryUsername, "", "Specify private docker registry username")
 	_ = cmd.PersistentFlags().String(installDockerRegistryPassword, "", "Specify private docker registry password")
+	_ = cmd.PersistentFlags().String(installAdminEmailParam, "", "Specify admin email (only for letsencrypt usage)")
```

Comment:
We should ask if a user wants to enable Let's Encrypt integration first

Reply:
Done